lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -100,6 +100,11 @@ Function Choco-IsInstalled
         [string]$package
     )
 
+   if ($package -like "all")
```

Comment:
I'm not very good at this open source/colab thing. 

Does it normally take 11 days for a trivial patch to get merged into master? 

If the syntactic sugar needs changing, just change it?

This bug is pretty shallow but waiting 11 days on the fix makes it feel really deep.